schema/base.graphqls

```diff
@@ -13,62 +20,45 @@ type Query {
   """
   signals(
     tokenID: Int!
+    """
+    interval is a time span that used for aggregatting the data with.
+    A duration string is a sequence of decimal numbers, each with optional fraction and a unit suffix,
+    such as "300ms", "-1.5h" or "2h45m". Valid time units are "ms", "s", "m", "h"
+    """
+    interval: String!
     from: Time!
     to: Time!
     filter: SignalFilter
-  ): SignalAggregations @requiresToken
-
+  ): [SignalAggregations!] @requiresToken
```

Comment:
Should this be `[SignalAggregations!]!`? Or are we allowed to return null and then some errors? Not a major point.